perma_web/perma/models.py

```diff
@@ -276,7 +282,7 @@ def is_registrar_member(self):
 
     def is_vesting_org_member(self):
         """ Is the user a member of a vesting org? """
-        return bool(self.vesting_org_id)
+        return bool(self.vesting_org.all())
```

Comment:
Should be more efficient to do `return self.vesting_org.exists()`

(I'm pretty sure that works -- otherwise `bool(self.vesting_org.count())`)